### ISSUE TITLE
Drop file access permission

### ISSUE THREAD

**Issue by oakkitten** [ubergeek42/weechat-android#511]
this is only used to load themes. import them instead. from dev console mail, dated yesterday

> Starting May 5th, you must let us know why your app requires broad storage access
> 
> We've detected that your app contains the `requestLegacyExternalStorage` flag in the manifest file of 1 or more of your app bundles or APKs.
> 
> Developers with apps on devices running Android 11+ must use Scoped Storage to give users better access control over their device storage. To release your app on Android 11 or newer after May 5th, you must either:
> 
> * Update your app to use more privacy friendly best practices, such as the Storage Access Framework or Media Store API
> * Update your app to declare the All files access (`MANAGE_EXTERNAL_STORAGE`) permission in the manifest file, and complete the All files access permission declaration in Play Console from May 5th
> * Remove the All files access permission from your app entirely
> * For apps targeting Android 11, the `requestLegacyExternalStorage` flag will be ignored. You must use the All files access permission to retain broad access.
> 
> Apps requesting access to the All files access permission without a permitted use will be removed from Google Play, and you won't be able to publish updates.

**Comment by oakkitten** [comment]
there's a small bug in current search branch; on android 11, turning phone off and on while the keyboard is up results in slightly incorrect window insets being applied. it looks like the behavior of `insets.systemWindowInsetTop` etc was changed in api 30. upon restoring activity, `mCompatInsetsTypes` doesn't contain the ime flag (`1<<3`)

the old behavior is possible via new API:

```kt
insets.getInsets(WindowInsets.Type.systemBars() or
                 WindowInsets.Type.navigationBars() or
                 WindowInsets.Type.ime())
```

the problem is, these api need targetting android 11, and that requires fixing this permission issue

as this is not a very significant bug, and only affects api 30, and because the fix for the permission issue is nearly there, let's postpone the fix until 1.7

**Comment by oakkitten** [comment]
file permission dropped
